src/components/data-management/LaunchAnalysisButton.jsx

```diff
@@ -1,31 +1,36 @@
 import React, { useState, useCallback, useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import { Button, Tooltip, Popconfirm } from 'antd';
+import {
+  Button, Tooltip, Popconfirm,
+} from 'antd';
 import { useRouter } from 'next/router';
 import moment from 'moment';
-import { updateExperimentInfo } from '../../redux/actions/experimentSettings';
+import { updateExperimentInfo } from 'redux/actions/experimentSettings';
 import {
   updateProject,
-} from '../../redux/actions/projects';
+} from 'redux/actions/projects';
 
-import fileUploadSpecifications from '../../utils/upload/fileUploadSpecifications';
-import UploadStatus from '../../utils/upload/UploadStatus';
-import pipelineStatus from '../../utils/pipelineStatusValues';
-import integrationTestConstants from '../../utils/integrationTestConstants';
-import generateGem2sParamsHash from '../../utils/data-management/generateGem2sParamsHash';
-import { runGem2s } from '../../redux/actions/pipeline';
-import { updateExperiment } from '../../redux/actions/experiments';
+import fileUploadSpecifications from 'utils/upload/fileUploadSpecifications';
+import UploadStatus from 'utils/upload/UploadStatus';
+import pipelineStatus from 'utils/pipelineStatusValues';
+import integrationTestConstants from 'utils/integrationTestConstants';
+import generateGem2sParamsHash from 'utils/data-management/generateGem2sParamsHash';
+import { runGem2s } from 'redux/actions/pipeline';
+import { updateExperiment } from 'redux/actions/experiments';
 
 const LaunchButtonTemplate = (props) => {
```

Comment:
Why is this split into a different component? you could move it into the `LaunchAnalysisButton` with a `renderButton()` or something, right?

I am not sure what is considered best practice, but I saw we do that in other places in the ui.

Reply:
This can certainly be written just as a function. It looks better as a React element though :P Anyways I'll move it into a function to make it simpler.

Reply:
Ok, we should probably decide on one way to do this and keep it in mind when working in other places.

If we are going with having auxiliary components as opposed to auxiliary render functions we should change the auxiliary render functions to components wehen we run into them in the future.